doc/documentation/python/include_yaml.py

```diff
@@ -0,0 +1,88 @@
+# This file is part of 4C multiphysics licensed under the
+# GNU Lesser General Public License v3.0 or later.
+#
+# See the LICENSE.md file in the top-level for license information.
+#
+# SPDX-License-Identifier: LGPL-3.0-or-later
+
+import yaml
+import pathlib
+import shutil
+import jinja2
+import argparse
+
+PATH_TO_TESTS = ""
+RECOGNIZED_TYPES = ["rst", "md"]
+
+
+def load_input_file(yaml_file):
+    data = yaml.safe_load((PATH_TO_TESTS / pathlib.Path(yaml_file)).read_text())
+    return data
+
+
+def yaml_dump(data, filetype="rst"):
+    if filetype not in RECOGNIZED_TYPES:
+        print(f"ERROR: Filetype {filetype} for yaml_dump cannot be recognized yet.")
+        return ""
+    if filetype == "rst":
+        rststring = ""
+        yaml_data = yaml.safe_dump(data, sort_keys=False).split("\n")
+        for line in yaml_data:
+            rststring += "    " + line + "\n"
+        rststring += "\n"
+        return ".. code-block:: yaml\n\n" + rststring + "\n"
+    elif filetype == "md":
+        return "```yaml\n" + yaml.safe_dump(data, sort_keys=False) + "```\n"
+    else:
+        return yaml.safe_dump(data, sort_keys=False)
```

Comment:
```suggestion
    if filetype == "rst":
        rststring = ""
        yaml_data = yaml.safe_dump(data, sort_keys=False).split("\n")
        for line in yaml_data:
            rststring += "    " + line + "\n"
        rststring += "\n"
        return ".. code-block:: yaml\n\n" + rststring + "\n"
    elif filetype == "md":
        return "```yaml\n" + yaml.safe_dump(data, sort_keys=False) + "```\n"
    else:
        raise TypeError(f"Filetype {filetype} for yaml_dump cannot be recognized yet.")
```